FlowCrypt/Controllers/Settings/Contacts/Contacts List/ContactKeyDetailViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ContactKeyDetailViewController.swift
+//  FlowCrypt
+//
+//  Created by Roma Sosnovsky on 13/10/21
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+    
+import AsyncDisplayKit
+import FlowCryptCommon
+import FlowCryptUI
+
+/**
+ * View controller which shows details about contact's public key
+ * - User can be redirected here from *ContactDetailViewController* by tapping on a particular key
+ */
+final class ContactKeyDetailViewController: TableNodeViewController {
+    typealias ContactKeyDetailAction = (Action) -> Void
+
+    enum Action {
+        case delete(_ key: ContactKey)
+    }
+
+    enum Part: Int, CaseIterable {
+        case key = 0, signature, created, checked, expire, longids, fingerprints, algo
+    }
+
+    private let decorator: ContactKeyDetailDecoratorType
+    private let key: ContactKey
+    private let action: ContactKeyDetailAction?
+
+    init(
+        decorator: ContactKeyDetailDecoratorType = ContactKeyDetailDecorator(),
+        key: ContactKey,
+        action: ContactKeyDetailAction?
+    ) {
+        self.decorator = decorator
+        self.key = key
```

Comment:
let's call it `publicKey` on both left and right side. So that it's obvious in the code when comparing to private keys.

Reply:
sure, pubKey is good